server/database.js

```diff
@@ -0,0 +1,12 @@
+const mysql = require('mysql');
+
+const DatabaseConnection = mysql.createConnection({
+  host: 'localhost',
+  database: 'fecimage',
+  user: 'root',
+  password: 'MattM',

```

Comment:
You should not hardcode local credentials. Use environment variables (`process.env`) instead.